src/GraphQL/Internal/AST.hs

```diff
@@ -1,9 +1,11 @@
 module GraphQL.Internal.AST
   ( Name
+  , pName
```

Comment:
Renamed to `nameParser`, as I would expect a function `parseName` to return a `Name` not a parser.